src/App.tsx

```diff
@@ -1,203 +1,161 @@
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
+import cn from 'classnames';
 import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import { getTodos, USER_ID } from './api/todos';
+import { Todo } from './types/Todo';
+import { TodoList } from './TodoList';
+import { Header } from './Header';
+import { Footer } from './Footer';
+import { Filter } from './types/Filter';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState([] as Todo[]);
```

Comment:
Do you need use `as` here?
```suggestion
  const [todos, setTodos] = useState<Todo[]>([]);
```